spyder/widgets/sourcecode/codeeditor.py

```diff
@@ -539,6 +539,10 @@ def create_shortcuts(self):
                                        name='Blockcomment', parent=self)
         unblockcomment = config_shortcut(self.unblockcomment, context='Editor',
                                          name='Unblockcomment', parent=self)
+        upper_selected_text = config_shortcut(self.text_uppercase, context='Editor',
+                                 name='Upper selected text', parent=self)
+        lower_selected_text = config_shortcut(self.text_lowercase, context='Editor',
+                                 name='Lower selected text', parent=self)

```

Comment:
Same here

`Transform selected text to uppercase`
`Transform selected text to lowercase`